src/Symfony/Bridge/Doctrine/Form/Type/EntityType.php

```diff
@@ -18,70 +18,17 @@
 class EntityType extends DoctrineType
 {
     /**
-     * @var ORMQueryBuilderLoader[]
-     */
-    private $loaderCache = array();
-
-    /**
      * Return the default loader object.
      *
-     * @param ObjectManager $manager
-     * @param mixed         $queryBuilder
-     * @param string        $class
+     * @param ObjectManager         $manager
+     * @param QueryBuilder|\Closure $queryBuilder
+     * @param string                $class
      *
      * @return ORMQueryBuilderLoader
      */
     public function getLoader(ObjectManager $manager, $queryBuilder, $class)
```

Comment:
IMO the $queryBuilder should be typehinted now. The closure is not passed to getLoader anymore, so for people relying on it, it would break anyway. Maybe we can change the signature now since people didn't overwrite the method probably.